src/lib/components/healthselectorv2/HealthSelector.style.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from 'styled-components';
+import Select from '../selectv2/Selectv2.component';
+import Icon from '../icon/Icon.component';
+import * as defaultTheme from '../../style/theme';
```

Comment:
we should likely used currently selected theme instead of the default one. 

To do so in styled component you can : ${props.theme[colorName]}.

Spacing can be imported directly through as it is not changing over selected theme